test/Test/Pos/CborSpec.hs

```diff
@@ -302,7 +303,7 @@ testAgainstFile name x expected =
               Right actual -> x `shouldBe` actual
 
 spec :: Spec
-spec = describe "Cbor.Bi instances" $ do
+spec = giveTestsConsts $ describe "Cbor.Bi instances" $ do
```

Comment:
what do you mean? do you suggest using `staticBlk...` everywhere besides `VarSpec` (and maybe some other tests)?